bciers/apps/registration/app/components/operations/registration/FacilityInformationForm.tsx

```diff
@@ -128,7 +128,18 @@ const FacilityInformationForm = ({
       // errors are handled in MultiStepBase
```

Comment:
```suggestion
```
Since it looks like you've moved error handling here

Reply:
Thanks, I just moved this comment to inside the error block. I didn't move error handling here, but it's important to check for an error and return it to MultiStepBase to handle if one occurs.